docs/DeveloperGuide.md

```diff
@@ -420,38 +486,81 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given student is invalid/cannot be found.
+* 3a. The inputted student's name is invalid or cannot be found.
 
     * 3a1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
 
-* 3b. Wrong format for deadline was given.
+* 3b. The inputted task deadline is invalid.
 
     * 3b1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
 
-* 3c. No task description was given.
+* 3c. No task description was inputted.
 
     * 3c1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
+
+***
+
+**Use case: View all tasks for all students**
+
+**MSS**
+1.  Tuition teacher requests to view all tasks for all students.
+2.  EduManage shows a list of students and their tasks.
+    
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
```

Comment:
May not be necessary? Unless you state an empty list is shown otherwise it's the same as MSS.